integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonInsertFromStageCommand.scala

```diff
@@ -499,25 +499,31 @@ case class CarbonInsertFromStageCommand(
    * return the loading files failed to create
    */
   private def createStageLoadingFiles(
+      stagePath: String,
       executorService: ExecutorService,
       stageFiles: Array[(CarbonFile, CarbonFile)]): Array[(CarbonFile, CarbonFile)] = {
     stageFiles.map { files =>
       executorService.submit(new Callable[(CarbonFile, CarbonFile, Boolean)] {
         override def call(): (CarbonFile, CarbonFile, Boolean) = {
-          // Get the loading files path
-          val stageLoadingFile =
-            FileFactory.getCarbonFile(files._1.getAbsolutePath +
-              CarbonTablePath.LOADING_FILE_SUFFIX);
-          // Try to create loading files
-          // make isFailed to be true if createNewFile return false.
-          // the reason can be file exists or exceptions.
-          var isFailed = !stageLoadingFile.createNewFile()
-          // if file exists, modify the lastModifiedTime of the file.
-          if (isFailed) {
-            // make isFailed to be true if setLastModifiedTime return false.
-            isFailed = !stageLoadingFile.setLastModifiedTime(System.currentTimeMillis());
+          try {
+            // Get the loading files path
+            val stageLoadingFile =
+              FileFactory.getCarbonFile(stagePath +
+                CarbonCommonConstants.FILE_SEPARATOR +
```

Comment:
Suggestion： use File.separator instead of CarbonCommonConstants.FILE_SEPARATOR， File.separator can support linux and window。

Reply:
modified